android/src/main/java/com/reactnative_stripeterminal/RNStripeTerminalModule.java

```diff
@@ -544,7 +544,7 @@ public void onSuccess(@Nonnull PaymentIntent paymentIntent) {
             @Override
             public void onFailure(@Nonnull TerminalException e) {
                 WritableMap errorMap = Arguments.createMap();
-                errorMap.putString(ERROR, e.getErrorMessage());
+                errorMap.putString(ERROR, e.getApiError().getMessage());
```

Comment:
Believe this should be added as a separate field since this may not always be provided (e.g. if error is not from the API).

Reply:
yes

Reply:
made a new PR with the fix